shell/platform/android/io/flutter/embedding/engine/FlutterShellArgs.java

```diff
@@ -37,6 +37,8 @@
   public static final String ARG_SKIA_DETERMINISTIC_RENDERING = "--skia-deterministic-rendering";
   public static final String ARG_KEY_TRACE_SKIA = "trace-skia";
   public static final String ARG_TRACE_SKIA = "--trace-skia";
+  public static final String ARG_KEY_DUMP_SHADER_SKP = "dump-shader-skp";
```

Comment:
Do these need to be pulled out of the `Intent` in `fromIntent()`?

Reply:
`FlutterActivityDelegate` is a mechanism in the old version of the embedding.  `FlutterShellArgs` is part of the new embedding.